canonical-web-and-design/snapcraft.io
# compress store badges

## Done

- compressed all Snapcraft store badges to ~35% of their original size

## QA

- checkout the master branch
- in a file browser, from the project root, navigate to /static/images/badges
- inspect the file sizes of any of the SVGs within, see that they are all 20-30kb
- checkout this branch
- in a file browser, from the project root, navigate to /static/images/badges
- inspect the file sizes of any of the SVGs within, see that they are all 10kb or less
